cmd/apmbench/bench.go

```diff
@@ -125,6 +125,7 @@ func Benchmark10000AggregationGroups(b *testing.B, l *rate.Limiter) {
 func newTracer(tb testing.TB) *apm.Tracer {
 	httpTransport, err := transport.NewHTTPTransport(transport.HTTPTransportOptions{
 		ServerURLs:  []*url.URL{loadgencfg.Config.ServerURL},
+		APIKey:      loadgencfg.Config.APIKey,
```

Comment:
This fixes `APIKey` auth for the apm-tracer based benchmarks but I haven't been able to find a way to add custom headers for these.